core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/SbomUtils.java

```diff
@@ -800,10 +802,10 @@ public static Bom fromJsonNode(JsonNode jsonNode) {
         }
     }
 
-    public static String[] computeNVRFromContainerManifest(JsonNode jsonNode) {
+    public static List<String> computeNVRFromContainerManifest(JsonNode jsonNode) {
         Bom bom = fromJsonNode(jsonNode);
         if (bom == null || bom.getComponents() == null || bom.getComponents().isEmpty()) {
-            return null;
+            return List.of();
```

Comment:
Converted from primitive array to `List`, but it may be better to have an `NVR` type.